browser/javascript/v2/react-app/src/ampli.test.js

```diff
@@ -0,0 +1,215 @@
+import { Ampli, ApiKey } from './ampli';
```

Comment:
Can you please update `.github/workflows` to run these tests in CI

Reply:
+1

Reply:
Added it into TS PR. Will merge TS example first before merging this one.